src/parsers/DocumentParser.js

```diff
@@ -23,7 +23,7 @@ export default class DocumentParser extends Parser {
 
         if (new.target === DocumentParser) {
             // Choose parser based on filename and language
-            if (isMatch('javascript', '**/.eslintrc.js')) {
+            if (isMatch('javascript', '**/.eslintrc.js', '**/.eslintrc.cjs')) {
```

Comment:
Thank you for this contribution.  There were other changes merged since this PR was created.  This will bring your change up to date with other changes:

```suggestion
            if (isMatch('javascript', '**/.eslintrc.js', '**/.eslintrc.cjs') || isMatch('javascriptreact', '**/.eslintrc.js', '**/.eslintrc.cjs')) {
```